src/db/mongo.rs

```diff
@@ -16,6 +16,10 @@ struct MongoMeigen {
     id: i64,
     author: String,
     content: String,
+
+    // Added in v0.3. The attribute is for the forward compatibility.
```

Comment:
:+1: nice documentation

Reply:
this is not request of change

Reply:
appreciation... ♡

Reply:
i did some mistake around here as we talked so [d314169](https://github.com/approvers/MeigenBot-Rust/pull/17/commits/d314169670f67138c58a6786e7fea206b413d60f)